refactoring-code/strangler-approach/1-create-scenarios.md

```diff
@@ -58,19 +59,29 @@ import
 
 Only add console.log statements. Do not change any other part of the code. Do not add any other code.
 
-## Step 3: Setup Standalone Environment
+## Step 3: Setup Standalone Environment // new chat window; otherwise started copying over rewrite directory
```

Comment:
we did this in a new context window because it started copying over `rewrite/` directory from step#1